index.js

```diff
@@ -80,6 +80,18 @@ mOcKiNgCaSe.overrideString = () => {
   return mOcKiNgCaSe;
 }
 
+/**
+ * Changes all console log output to mOcKiNgCaSe.
+ *
+ */
+mOcKiNgCaSe.overrideConsole = (options = {}) => {
+  let original = console.log;
```

Comment:
better to use const if the variable is not reassigned
```suggestion
  const original = console.log;
```